src/generator/predefined/generator.js

```diff
@@ -0,0 +1,119 @@
+const createTemplate = module.parent.createTemplate;
+
+const generatorTemplate = createTemplate`
+/* DO NOT MODIFY */
+const createTemplate = module.parent.createTemplate;
+/* END OF DO NOT MODIFY */
+
+/**
+ * To define file template use createTemplate tag function.
+ * You can define multiple templates and for example conditionally apply them to entry:
+ * template: flag ? template0 : template1
+ * You can also use interpolation \${} and put any valid JS expression.
+ * To render value based on arguments use arrow function:
+ * \${args => args.value}
+ */
+const template = createTemplate\`
+  const var0 = \${Math.pow(2, 2)};
+  const var1 = '\${args => args.var1}';
+\`;
+
+
+/**
+ * Generator must export object with configuration or function that returns
+ * object with configuration.
+ * If export is a function, it will receive the following object as first argument:
+ * {
+ *   name: string; // name of entity specified on generate command execution
+ *   generator: string; // generator name eg: component, reducer, container
+ * }
+ */
+module.exports = exports = options => ({
+  /**
+   * Define shared arguments.
+   * Those arguments that are passed down to every entry.
+   * Those arguments can be overwritten by entry-specific arguments.
+   * Type: Object
+   */
+  args: {
+    var: JSON.stringify(options)
+  },
+  /**
+   * Define modificator to alter existing files or create new one.
```

Comment:
There are a handful of places that use the word `modificator`. I think we really want `modifier`?